tests/test_connection.py

```diff
@@ -22,6 +22,9 @@
 import unittest
 import websocket
 
+# Big enough to make channels with 10k effective capacity, including Elements channels
+# which have bigger txns
+CHANNEL_SIZE = 50000
```

Comment:
maybe we can move this in the `tests/utils.py`?

Reply:
done